requirements/dev.in

```diff
@@ -29,8 +29,8 @@ isort
 # For doing highly usable Python profiling
 line-profiler
 
-# for pep8 linter
-pycodestyle
+# Python reformatter
+black
```

Comment:
Hmm, I suppose if all pycodestyle does is checks formatting, this would be a correct replacement.